web/docs/tql2/operators.md

```diff
@@ -0,0 +1,152 @@
+# Operators
+
+import './operators.css';
+
+## Basic
+
+Operator | Description | Example
+---------|-------------|--------
+[`drop`](operators/drop) | Removes columns from the event | `drop name, metadata.id`
+[`head`](operators/head) | Keep only the first N events | `head 20`
+[`select`](operators/select) | Selects values and discards the rest | `select name, id=metadata.id`
+[`set`](operators/set) | Explicitly assigns one or multiple values | `set name="Tenzir", id=metadata.id`
```

Comment:
Isn't this just `name = "Tenzir" | id = metadata.id`?

Reply:
Yes. The `set` operator can almost always be directly replaced with multiple assignments.